nextflow_schema.json

```diff
@@ -568,7 +568,7 @@
             "properties": {
                 "max_cpus": {
```

Comment:
Missing the new flags in the main schema?

Reply:
Just pushed :)

Reply:
You mean the `mime-type` stuff right? If so, then yes.